[BUG] Cannot import dashboard

**Describe the bug**
When trying to import a dashboard json-file, nothing happens when clicking "Import".

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '/index/dashboard/'
2. Click on the gear-sign at the right and export a dashboard
3. Start a new browser with locale explicitly set to C, for instance: ```$ LC_ALL=C firefox```
4. Go to '/index/dashboard/' and click the plus-sign to the left, just right of "Dashboards:"
5. Click on "Import dashboard"
6. Select the file created in step 2
3. Click "Import"
4. Nothing happens! You can click import forever.

**Expected behavior**
The dashboard should be imported and be selectable.

**Environment (please complete the following information):**
 - OS on the NAV server: Debian stretch
 - NAV version installed:  5.0.4
 - Method of installation: from Debian package

Error-mail with exception received, recreated bug locally:

```
web_1       | [Thu Mar 12 13:13:36 2020] [ERROR] [pid=281 django.request] Internal Server Error: /index/dashboard/import
web_1       | Traceback (most recent call last):
web_1       |   File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/exception.py", line 41, in inner
web_1       |     response = get_response(request)
web_1       |   File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 187, in _get_response
web_1       |     response = self.process_exception_by_middleware(e, request)
web_1       |   File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 185, in _get_response
web_1       |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
web_1       |   File "/usr/local/lib/python3.5/dist-packages/django/views/decorators/http.py", line 40, in inner
web_1       |     return func(request, *args, **kwargs)
web_1       |   File "/source/python/nav/web/webfront/views.py", line 128, in import_dashboard
web_1       |     data = json.load(request.FILES['file'])
web_1       |   File "/usr/lib/python3.5/json/__init__.py", line 268, in load
web_1       |     parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
web_1       |   File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
web_1       |     s.__class__.__name__))
web_1       | TypeError: the JSON object must be str, not 'bytes'
```

--- a/python/nav/web/webfront/views.py
+++ b/python/nav/web/webfront/views.py
@@ -125,7 +125,9 @@ def import_dashboard(request):
     response = {}
     if 'file' in request.FILES:
         try:
-            data = json.load(request.FILES['file'])
+            # Ensure file is interpreted as utf-8 regardless of locale
+            blob = request.FILES['file'].read()
+            data = json.loads(blob.decode("utf-8"))
             if not isinstance(data, dict):
                 raise ValueError()
             for field, dtype in dashboard_fields.items():

